dts/bindings/flash_controller/jedec,ospi-nor.yaml

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) 2021, STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    STM32 OSPI Flash controller supporting the JEDEC CFI interface
+
+    Representation of a serial flash on a octospi bus:
+
+        mx25lm51245: ospi-nor-flash@0 {
+                compatible = "st,stm32-ospi-nor";
+                label = "MX25LM512";
+                reg = <0>;
+                data-mode = <OSPI_OPI_MODE>; /* access on 8 data lines */
+                data-rate = <OSPI_DTR_TRANSFER>; /* access in DTR */
+                ospi-max-frequency = <DT_FREQ_M(50)>;
```

Comment:
Not valid probably missed to update, the property is now renamed to `spi-max-frequency`